src/m365/entra/commands/m365group/m365group-user-add.ts

```diff
@@ -81,6 +102,14 @@ class EntraM365GroupUserAddCommand extends GraphCommand {
           return `${args.options.groupId} is not a valid GUID`;
         }
 
+        if (args.options.ids && args.options.ids.split(',').some(e => !validation.isValidGuid(e))) {
+          return `${args.options.ids} contains one or more invalid GUIDs`;
+        }
+
+        if (args.options.userNames && args.options.userNames.split(',').some(e => !validation.isValidUserPrincipalName(e))) {
+          return `${args.options.userNames} contains one or more invalid usernames`;
+        }
```

Comment:
Let's use both `validation.isValidGuidArray` and `validation.isValidUserPrincipalNameArray` for this.